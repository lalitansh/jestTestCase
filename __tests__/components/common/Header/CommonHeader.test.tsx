import renderer from 'react-test-renderer';
import React from 'react';
import CommonHeader from '../../../../src/components/common/Header/commonHeader';

it('renders correctly across screens', () => {
  const tree = renderer.create(<CommonHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});