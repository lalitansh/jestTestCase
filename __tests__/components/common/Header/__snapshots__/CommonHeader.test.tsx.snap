// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly across screens 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#167351",
      "flexDirection": "row",
      "height": 65,
      "marginTop": undefined,
      "paddingHorizontal": 20,
      "paddingVertical": 16,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flex": 0.2,
      }
    }
  />
  <View
    style={
      Object {
        "flex": 0.8,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 16,
          },
          Object {
            "textAlign": "center",
          },
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "flex": 0.2,
        },
        Object {
          "alignItems": "flex-end",
        },
      ]
    }
  />
</View>
`;
